Nim/Program.cs

```diff
@@ -0,0 +1,441 @@
+﻿using System.Text;
+
+namespace NimConsoleApplication
+{
+    public static class Program
+    {
+        private static int _winCountBasicMenu; private static int _lostCountBasicMenu;
+        private static int _winCountFunMenu; private static int _lostCountFunMenu;
+
+        static void Main()
```

Comment:
You can safe yourself the trouble of defining a static class with a static main method and static fields since .NET 5.


Instead of

```c#
namespace X{
   static class Foo{
      static int a;
      static void Main(){
          a++;
       }
   }
}
```

You can simply do:

```c#
int a;
a++;
```

(compiles to nearly the same code)